accounts/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from .forms import UserForm
+from django.contrib.auth import authenticate, login, logout
+
+
+# Create your views here.
+def register_page(request):
+    """Register user account for the site."""
+    form = UserForm()
+
+    if request.method == "POST":
+        form = UserForm(request.POST)
+        if form.is_valid():
+            form.save()
+            user = form.cleaned_data.get('username')
+            messages.success(request, f'Account for {user} was created')
+            return redirect('login')
+        else:
+            messages.info(request, 'Username or Password is incorrect.')
+    context = {'form': form}
+    return render(request, 'accounts/register.html', context)
+
+
+def login_page(request):
+    """Log user into the site."""
+    if request.method == "POST":
+        username = request.POST.get('username')
+        password = request.POST.get('password')
+
+        user = authenticate(request, username=username, password=password)
+
+        if user is not None:
+            login(request, user)
+            return redirect('events:feed')
+        else:
+            messages.warning(request, 'Username or Password is incorrect.')
+
+    context = {}
+    return render(request, 'accounts/login.html', context)
```

Comment:
There is no context, so you don't need to pass it through the template.

Reply:
fixed it!